.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.0
 jobs:
     build:
         docker:
-            - image: circleci/node:8
+            - image: circleci/node:10
```

Comment:
Any reason to stop at 10? The current LTS appears to be 14

src: https://nodejs.org/en/

Reply:
Only that I hadn't realised there were newer versions.

Reply:
```suggestion
            - image: circleci/node:14
```